src/server/server.test.ts

```diff
@@ -1,15 +1,20 @@
-import axios from 'axios';
-import { Server } from 'http';
-import { AddressInfo } from 'net';
-import { randomTranscript } from '../types/gentrans';
-import { app } from './server';
-import { Transcript } from '../types/transcript';
-import { initialize, addStudent, getTranscript, getGrade } from '../types/local-transcript-manager';
+import axios from "axios";
+import { Server } from "http";
+import { AddressInfo } from "net";
+import { randomTranscript } from "../types/gentrans";
+import { app } from "./server";
+import { Transcript } from "../types/transcript";
+import {
+  initialize,
+  addStudent,
+  getTranscript,
+  getGrade,
+} from "../types/local-transcript-manager";
 
-describe('TranscriptREST', () => {
-  let server : Server;
-  let baseurl : string;
-  beforeAll( () => {
+describe("TranscriptREST", () => {
```

Comment:
Preferable not to change the styling and code irrelevant to the issue

Reply:
I agree, please stick to the styling being used